nibabel/cifti2/cifti2.py

```diff
@@ -1484,8 +1485,14 @@ def update_headers(self):
         >>> img.update_headers()
         >>> img.nifti_header.get_data_shape() == (2, 3, 4)
```

Comment:
```suggestion
        >>> img.nifti_header.get_data_shape() == (1, 1, 1, 1, 2, 3, 4)
```

Reply:
this breaks the current API, is that fine?

Reply:
In what sense? Does it change `img.shape`? If so, we should divorce `Cifti2Image.shape` from `Cifti2Image.nifti_header.get_data_shape()`.